firefox-ios/firefox-ios-tests/Tests/XCUITests/ToolbarMenuTests.swift

```diff
@@ -58,22 +58,22 @@ class ToolbarMenuTests: BaseTestCase {
             "Menu button is not below the pocket cells area"
         )
         hamburgerMenu.tap()
-        mozWaitForElementToExist(app.images[StandardImageIdentifiers.Large.avatarCircle])
+        mozWaitForElementToExist(app.buttons[AccessibilityIdentifiers.MainMenu.HeaderView.mainButton])
         validateMenuOptions()
         app.buttons["MainMenu.CloseMenuButton"].tap()
-        mozWaitForElementToNotExist(app.images[StandardImageIdentifiers.Large.avatarCircle])
+        mozWaitForElementToNotExist(app.buttons[AccessibilityIdentifiers.MainMenu.HeaderView.mainButton])
     }
 
     private func validateMenuOptions() {
         waitForElementsToExist(
             [
-                app.images[StandardImageIdentifiers.Large.avatarCircle],
-                app.images[StandardImageIdentifiers.Large.plus],
-                app.images[StandardImageIdentifiers.Large.privateModeCircleFill],
-                app.images[StandardImageIdentifiers.Large.bookmarkTrayFill],
-                app.images[StandardImageIdentifiers.Large.history],
-                app.images[StandardImageIdentifiers.Large.download],
-                app.images[StandardImageIdentifiers.Large.login]
+                app.buttons[AccessibilityIdentifiers.MainMenu.HeaderView.mainButton],
+                app.tables.cells[AccessibilityIdentifiers.MainMenu.newTab],
+                app.tables.cells[AccessibilityIdentifiers.MainMenu.newPrivateTab],
+                app.tables.cells[AccessibilityIdentifiers.MainMenu.bookmarks],
+                app.tables.cells[AccessibilityIdentifiers.MainMenu.history],
+                app.tables.cells[AccessibilityIdentifiers.MainMenu.downloads],
+                app.tables.cells[AccessibilityIdentifiers.MainMenu.passwords]
```

Comment:
On iOS 15 and 16, the images don't have identifiers. We need to use the labels to identify the menu items.